src/EnergyPlus/DXCoils.cc

```diff
@@ -13811,8 +13811,7 @@ void CalcMultiSpeedDXCoilHeating(EnergyPlusData &state,
 
                 if (FractionalDefrostTime > 0.0) {
                     // Calculate defrost adjustment factors depending on defrost control strategy
-                    if (thisDXCoil.DefrostStrategy == StandardRatings::DefrostStrat::ReverseCycle &&
-                        thisDXCoil.DefrostControl == StandardRatings::HPdefrostControl::OnDemand) {
+                    if (thisDXCoil.DefrostStrategy == StandardRatings::DefrostStrat::ReverseCycle) {
```

Comment:
This look right to me. If any diffs pop up, check the inputs.

Reply:
I am documenting the defrost code from CalcDXHeatingCoil so that it is clear that these models use the same defrost code. It seems to me that these are now using the same model/conditionals/etc.

        // Check outdoor temperature to determine of defrost is active
        if (OutdoorDryBulb <= thisDXCoil.MaxOATDefrost && thisDXCoil.CondenserType(Mode) != DataHeatBalance::RefrigCondenserType::Water) {
            // Calculate defrost adjustment factors depending on defrost control type
            if (thisDXCoil.DefrostControl == StandardRatings::HPdefrostControl::Timed) {
                FractionalDefrostTime = thisDXCoil.DefrostTime;
                if (FractionalDefrostTime > 0.0) {
                    HeatingCapacityMultiplier = 0.909 - 107.33 * OutdoorCoildw;
                    InputPowerMultiplier = 0.90 - 36.45 * OutdoorCoildw;
                }
            } else { // else defrost control is on-demand
                FractionalDefrostTime = 1.0 / (1.0 + 0.01446 / OutdoorCoildw);
                HeatingCapacityMultiplier = 0.875 * (1.0 - FractionalDefrostTime);
                InputPowerMultiplier = 0.954 * (1.0 - FractionalDefrostTime);
            }

            if (FractionalDefrostTime > 0.0) {
                // Calculate defrost adjustment factors depending on defrost control strategy
                if (thisDXCoil.DefrostStrategy == StandardRatings::DefrostStrat::ReverseCycle) {
                    LoadDueToDefrost = (0.01 * FractionalDefrostTime) * (7.222 - OutdoorDryBulb) * (thisDXCoil.RatedTotCap(Mode) / 1.01667);
                    DefrostEIRTempModFac = CurveValue(state, thisDXCoil.DefrostEIRFT, max(15.555, InletAirWetBulbC), max(15.555, OutdoorDryBulb));
                    thisDXCoil.DefrostPower = DefrostEIRTempModFac * (thisDXCoil.RatedTotCap(Mode) / 1.01667) * FractionalDefrostTime;
                } else { // Defrost strategy is resistive
                    thisDXCoil.DefrostPower = thisDXCoil.DefrostCapacity * FractionalDefrostTime;
                }
            } else { // Defrost is not active because (FractionalDefrostTime .EQ. 0.0)
                thisDXCoil.DefrostPower = 0.0;
            }
        }